step.yml

```diff
@@ -124,6 +124,16 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - auto_merge: "yes"
+    opts:
+      category: Debug
+      title: Merge origin/master
+      summary:  Merge origin/master 
+      description: |-
+        Merge origin/master always
```

Comment:
Master is not always the PR target, description should say that this flag merges our source branch into the destination branch.

Reply:
Switching this flag to `true` indicates that the source branch should be merged into the destination branch. This is useful if you want to build/test the `integrated state` (e. g. feature branch merged into master) instead of just the source branch.